posts/2020-06-22-jakarta-ee-9-open-liberty-beta.adoc

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "Jakarta EE beta on Open Liberty"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/nottycode
+author_github: https://github.com/nottycode
+seo-title: Jakarta EE beta on Open Liberty - OpenLiberty.io
+seo-description: DESCRIPTION
+blog_description: "DESCRIPTION"
+additional_authors: 
+- name: Kevin Sutter
+  github: https://github.com/kwsutter
+  image: https://avatars0.githubusercontent.com/kwsutter
+---
+= Jakarta EE beta on Open Liberty
+Alasdair Nottingham <https://github.com/nottycode>
+
+Our first Open Liberty beta (released today) makes available the first three Jakarta EE 9 features that we've implemented. We'll be releasing betas every four weeks and adding more Jakarta EE 9 features as we go.
+
+== Open Liberty betas
+
+When we first released the Open Liberty open source project, we focused mainly on adding production-quality capabilities to Liberty and releasing completed features. While we have often talked about our in-development work so that you know what's coming, the only way to get hold of this unfinished code to it try out was to take a look at our nightly builds. This is never ideal because nightly builds can be somewhat unstable, so we wanted to release a regular Open Liberty beta. 
+
+Open Liberty betas will contain everything in the most recent Open Liberty release plus features that are not yet finished. While these features are not yet suitable for use in production, we want to get feedback from our user community and them available to developers to use ahead of a production release. Of course, something appearing in a beta isn't a guarantee that it'll end up released: the feedback might be that it is a horrible, terrible, no good, very bad idea, and should be constrained to the dumpster of git history. At the very least, it might change a bit.
```

Comment:
I think the word make is missing from the following. Also to developers to doesn't sound write so I made a suggestion.

>While these features are not yet suitable for use in production, we want to get feedback from our user community and _make_ them available to developers ~to~ _for_ use ahead of a production release.

Reply:
thx